setup.py

```diff
@@ -41,7 +41,7 @@
         # certifi has no version requirement
         # CLI raises a warning for outdated package instead
         "certifi",
-        "cookiecutter==1.7.2",
+        "cookiecutter>=0.9.0,<=1.7.1",
```

Comment:
Please don't use the upper cap.
Upper limits should be used in `pip install --constraint FILE ...` instruction, not in the library requirements.

Reply:
sure. actually, we don't even need the upper constraint.

Reply:
Great!